src/publish-registry.ts

```diff
@@ -152,11 +152,13 @@ function generatePackageJson(version: string, typesPublisherContentHash: string)
 	};
 }
 
-interface Registry { readonly entries: { readonly [packageName: string]: 1 }; }
-function generateRegistry(typings: ReadonlyArray<TypingsData>): Registry {
-	const entries: { [packageName: string]: 1 } = {};
-	for (const { name } of typings) {
-		entries[name] = 1;
+interface Registry { readonly entries: { readonly [packageName: string]: { readonly [distTags: string]: string } }; }
+async function generateRegistry(typings: ReadonlyArray<TypingsData>): Promise<Registry> {
+	const entries: { [packageName: string]: { [distTags: string]: string } } = {};
+	await nAtATime(25, typings, getTags);

```

Comment:
Personally I prefer to write just `await nAtATime(25, typings, async typing => { ... })` with no function name or explicit type annotation.